google-cloud-clients/google-cloud-contrib/README.md

```diff
@@ -12,6 +12,7 @@ Packages that provide higher-level abstraction/functionality for common Google C
 Contents
 --------
 
+ * [google-cloud-logging](./google-cloud-logging) Logback appender for Google Cloud Logging.
```

Comment:
Shouldn't this be `google-cloud-logging-logback`?

Reply:
yep, done